### ISSUE TITLE
USB bluetooth..

### ISSUE THREAD

**Issue by dezral** [happy-bubbles/presence#19]
could this be made to use USB bluetooth? i run my nodered/homebridge on a rpi2, and have bought this "Bus 001 Device 004: ID 0b05:17cb ASUSTek Computer, Inc. Broadcom BCM20702A0 Bluetooth" it supports ble and can find my devices via "sudo hcitool lescan" but need to get the information into node-red using MQTT would be nice..

**Comment by nemik** [comment]
Yes, it could. I'd recommend writing or finding a script or program that can use the RPI to scan for BLE devices. Then if it sends what it finds over MQTT in the same format, to the same topics, to the same broker, that the Happy Bubbles hardware does, you'd have what you want. The presence server would treat it just like the Happy Bubbles hardware detector.
